website/docs/general/contributor-guide.md

```diff
@@ -58,55 +54,83 @@ Good First Issue curates easy pickings from this project, and helps you make you
 - [Apache APISIX® Java Plugin Runner](https://github.com/apache/apisix-java-plugin-runner/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
 - [Apache APISIX® Go Plugin Runner](https://github.com/apache/apisix-go-plugin-runner/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
 
-### Create your branch
+## Git Flow
+
+To contribute code or documentation, setup your local machine for development.
 
-Switch to your forked master branch, pull codes from upstream, then create a new branch.
+### Fork the repo and clone your fork
+
+Fork the [Apache APISIX repo](https://github.com/apache/apisix/) and clone your fork to your local machine.
 
 ```sh
-$ git checkout master
-$ git pull upstream master
-$ git checkout -b IssueNo
+git clone https://github.com/your-username/apisix.git
 ```
 
-Notice: We will merge PR using squash, commit logs will be different form upstream if you use one older branch.
-
-### Coding
+### Add 'upstream' repo to list of remotes
```

Comment:
If we are using forked repo then the upstream will be set already. It would be good to write about check and add.